.zenodo.json

```
@@ -0,0 +1,107 @@
+{
+  "creators": [
+    {
+      "name": "Tal Yarkoni"
+    },
+    {
+      "affiliation": "University of Texas at Austin",
+      "name": "Alejandro de la Vega",
+      "orcid": "0000-0001-9062-3778"
+    },
+    {
+      "affiliation": "Stanford University",
+      "name": "Christopher J. Markiewicz",
+      "orcid": "0000-0002-6533-164X"
+    },
+    {
+      "affiliation": "Department of Psychology, Stanford University",
+      "name": "Chris Filo Gorgolewski",
+      "orcid": "0000-0003-3321-7583"
+    },
+    {
+      "affiliation": "Florida International University",
+      "name": "Taylor Salo",
+      "orcid": "0000-0001-9813-3167"
+    },
+    {
+      "name": "Krista DeStasio"
+    },
+    {
+      "affiliation": "Dartmouth College",
+      "name": "Yaroslav O. Halchenko",
+      "orcid": "0000-0003-3456-2493"
+    },
+    {
+      "name": "Dmitry Petrov"
+    },
+    {
+      "affiliation": "National Institute of Mental Health",
+      "name": "Dylan Nielson",
+      "orcid": "0000-0003-4613-6643"
+    },
+    {
+      "name": "Kirstie Whitaker",
+      "orcid": "0000-0001-8498-4059"
+    },
+    {
+      "affiliation": "Montreal Neurological Institute and Hospital",
+      "name": "Elizabeth DuPre",
+      "orcid": "0000-0003-1358-196X"
+    },
+    {
+      "name": "Greg Kiar"
+    },
+    {
+      "affiliation": "CorticoMetrics LLC",
+      "name": "Lee S. Tirrell",
+      "orcid": "0000-0001-9393-8361"
+    },
+    {
+      "name": "Alexander Ivanov"
+    },
+    {
+      "affiliation": "Otto-von-Guericke-University Magdeburg, Germany",
+      "name": "Michael Hanke",
+      "orcid": "0000-0001-6398-6370"
+    },
+    {
+      "affiliation": "T\u00e9l\u00e9com ParisTech, France",
```

yikes what happened here. Let's just do:

```suggestion
      "affiliation": "Telecom ParisTech, France",
```

without getting too pedantic about French accents ...

---

The `json` module by default converts everything to ASCII. It renders fine. See, e.g. Stadler, Jörg in the [Nipype entry](https://zenodo.org/record/1560596). This is produced by [this line](https://github.com/nipy/nipype/blob/0fde422d0a6fdde289aace4bb7f24c16f8bf662e/.zenodo.json#L372).

We can probably avoid forcing to ASCII, but it hasn't caused problems in the past.

---

okay fair enough!